krynn/krynn.php

```diff
@@ -134,7 +136,7 @@ function krynn_settings_post($a,$post) {
 
 
 
-function krynn_settings(&$a,&$s) {
+function krynn_settings(App $a, $s) {
```

Comment:
The `&` operator of the `$s` parameter enables this function to change the hook data. If you remove it this function has no effect whatsoever.